ledger/participant-integration-api/src/main/scala/platform/index/ReadOnlySqlLedger.scala

```diff
@@ -135,9 +141,18 @@ private[platform] object ReadOnlySqlLedger {
     def contractsStoreOwner(
         lfValueTranslationCache: LfValueTranslationCache.Cache,
         contractsReader: LedgerDaoContractsReader,
-    ): Resource[ContractStore] =
-      TranslationCacheBackedContractStore
-        .owner(lfValueTranslationCache, contractsReader)
+    )(implicit executionContext: ExecutionContext): Resource[ContractStore] =
+      if (enableMutableContractStateCache)
+        MutableCacheBackedContractStore.owner(
+          contractsReader = contractsReader,
+          signalNewLedgerHead = _ => (), // TODO mutable contract state cache: wire up dispatcher
```

Comment:
Are you planning to do this in this PR or a follow-up?

Reply:
We've decided to implement this in a follow-up PR in order to allow E2E integration now.